setup.py

```diff
@@ -1668,6 +1668,20 @@ class db_found(Exception): pass
         if '_tkinter' not in [e.name for e in self.extensions]:
             missing.append('_tkinter')
 
+        # Build the _uuid module if possible
+        build_uuid = False
+        if find_file("uuid.h", inc_dirs, ["/usr/include/uuid"]):
+            if self.compiler.find_library_file(lib_dirs, 'uuid'):
```

Comment:
You may also store the result of find_library_file() into uuid_libdirs to pass it to Extension library_dirs. I'm not sure about this one.

Reply:
Let me try...

Reply:
No, it shouldn't be necessary. Other uses of `find_library_file()` don't do it.

Reply:
Ok, fine.